src/NzbDrone.Core/MediaFiles/DiskScanService.cs

```diff
@@ -67,26 +67,29 @@ public void Scan(Series series)
         {
             var rootFolder = _diskProvider.GetParentFolder(series.Path);
 
-            if (!_diskProvider.FolderExists(rootFolder))
+            var rootFolderExists = _diskProvider.FolderExists(rootFolder);
+            if (!rootFolderExists)
             {
                 _logger.Warn("Series' root folder ({0}) doesn't exist.", rootFolder);
-                _eventAggregator.PublishEvent(new SeriesScanSkippedEvent(series, SeriesScanSkippedReason.RootFolderDoesNotExist));
+                _eventAggregator.PublishEvent(new SeriesScanSkippedEvent(series,
```

Comment:
This is just a formatting thing that my IDE did automatically, the rest of the line is on the line below, I'll undo that.